### PR TITLE
Awesome book plain JavaScrupt

### PR DESCRIPTION
**Pull request by Errytagedesign**
In this branch, we did:

- Create a add book functions
- Create saveBook function to persist data to localstorage
- Create a remove book function to filter out and delete books from collections
- Render books in the collections to UI dynamically

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Errytagedesign, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses portfolio readme, kindly adapt it to this very project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by souad988 — COMMENTED** [review]
Hi @Errytagedesign , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is meant to be a pair programming activity but I see only commits from one student please reach out to your peer and ask them to do their part of work 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+/* eslint-disable no-unused-vars */
+function removeBook(id) {
+  const indexOfBookToRemove = bookCollections.filter((book) => book.id === id);
+
+  if (indexOfBookToRemove !== -1) {
+    bookCollections.splice(indexOfBookToRemove, 1);
+    saveBooks();
+    renderBooks();
+  }
+}
```
Comment (wayungi) [inline app.js]: - [ ] Your delete book function does not delete the selected book. The function simply deletes the books inorder of oldest to newest. ie, if you add book1,book2, ..., book4 and delete book4, it is book1 that will be deleted. Please add a unique Id to all books and use the id to target the book to be removed. 

Your books are not stored with an Id currently
id

File: README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+- GitHub: [@githubhandle](https://github.com/Errytagedesign)
+- Twitter: [@twitterhandle](https://twitter.com/errytage)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/errytagedesigns/)
+
+👤 Abu Talha
+
+- GitHub: [@githubhandle](https://github.com/AbuTalha3)
+- Twitter: [@twitterhandle](https://twitter.com/errytage)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/errytagedesigns/)
```
Comment (wayungi) [inline README.md]: - [ ] These links lead to the wrong person. They are supposed to lead to AbuTahlha3's social sites. Please have them fix or deleted if AbhuTalha does not have these social accounts. I would suggest that he creates them. 👍🏾

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Errytagedesign  & @AbuTalha3 , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks passing ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Errytagedesign and @AbuTalha3 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**